.travis.yml

```diff
@@ -95,7 +103,7 @@ install:
     - ls -la /home/travis/build
     - ls -la /home/travis/builds
     # this should always point to the newest version of GSL we support
```

Comment:
No, actually we should leave this alone. GSL 2.2.1 is the most recent released version of GSL that our code is known to work with. If we use the GSL master branch here, a bug will cause all builds on all GSL versions to fail, when really we would like to see that only GSL master has a problem.